src/Common/src/CoreLib/System/HashCode.cs

```diff
@@ -69,7 +69,12 @@ public struct HashCode
         private static unsafe uint GenerateGlobalSeed()
         {
             uint result;
+#if !BUILDINGNETSTANDARD20
```

Comment:
@stephentoub, I suggested to @joperezr to add a stub similar to the `BitOperations`. He mentioned you told him to `#if` instead. Not that i matters fro the customer, but I was just curious for the rationale. I thought our guidance is generally to prefer stubs when possible.

Reply:
If I said to prefer if/else/endif, I didn't mean it :)  I'd also prefer a partial file.

Reply:
It would be best to add stub implementation of `Interop.GetRandomByte` and leave this file alone.

Reply:
Ah, yeah, that'd be better.

Reply:
fixed.